test/index.test.ts

```diff
@@ -416,6 +417,62 @@ describe('run', function () {
     })
   }) // end 'target'
 
+  describe('distTag as target', () => {
+    /** Compact function to test ncu */
+    const testNcuTargetUpgrade = async (target: Target, currentVersion: string, upgradeVersion: string) => {
```

Comment:
My preference for unit tests is to use low-level, literal statements when possible and avoid creating abstractions. This makes each test self-contained and saves developers from having to learn a special API to read the tests.  This contrasts with my priorities in production code, where I tend to value abstraction and the elimination of redundancies. I only use abstractions in test code when the low-level code is so convoluted as to make it hard to read. 

Please inline the test code and remove the `testNcuTargetUpgrade` function.

Reply:
done